ska_helpers/chandra_models.py

```diff
@@ -26,6 +29,48 @@
     "https://api.github.com/repos/sot/chandra_models/releases/latest"
 )
 
+ENV_VAR_NAMES = [
+    "CHANDRA_MODELS_REPO_DIR",
+    "CHANDRA_MODELS_DEFAULT_VERSION",
+    "THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW",
+]
+
+
+def chandra_models_cache(func):
+    """Decorator to cache outputs for a function that gets chandra_models data.
+
+    The key used for caching the function output includes the passed arguments and
+    keyword arguments, as well as the values of the environment variables below.
+    This ensures that the cache is invalidated if any of these environment variables
+    change:
+
+    - CHANDRA_MODELS_REPO_DIR
+    - CHANDRA_MODELS_DEFAULT_VERSION
+    - THERMAL_MODELS_DIR_FOR_MATLAB_TOOLS_SW
+
+    Example::
+
+        @chandra_models_cache
+        def get_aca_spec_info(version=None):
+            _, info = get_data("chandra_models/xija/aca/aca_spec.json", version=version)
+            return info
+    """
+    cache = LRUDict(capacity=32)
```

Comment:
Oh, and why 32?

Reply:
I figure we'll never have more than 32 chandra_models data files that get read in. I was worried about making it too big in case we have some big files. But it could be 64 or 128.